percona/controllers/version/version.go

```diff
@@ -13,49 +14,86 @@ import (
 	api "github.com/percona/percona-postgresql-operator/pkg/apis/crunchydata.com/v1"
 )
 
-func EnsureVersion(clientset kubeapi.Interface, cr *api.PerconaPGCluster, vs VersionService) error {
-	if cr.Spec.UpgradeOptions == nil ||
-		cr.Spec.UpgradeOptions.Apply.Lower() == api.UpgradeStrategyNever ||
-		cr.Spec.UpgradeOptions.Apply.Lower() == api.UpgradeStrategyDisabled {
-		return nil
-	}
+const EnvDisableTelemetry = "DISABLE_TELEMETRY"
```

Comment:
I don't see this public const used anywhere besides the `TelemetryEnabled` func, maybe have it as a local const inside that func.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/273/commits/ba795d21f236b4a0da4a70d7c7bb197abcdd3e27